src/modules/selector.ts

```diff
@@ -1,5 +1,5 @@
 import {dfuProgrammerFlash} from './programmers/dfu-programmer';
-import {stm32, kiibohd} from './programmers/dfu-util';
+import {Family, dfuUtilFlash} from './programmers/dfu-util';
```

Comment:
I'm not sure I like the use of this being called Family as this is not the only programmer with more than 1 bootloader with different commands, e.g avrdude has a number. I think it should someway incorporate to tell us it's dfuUtil things

Reply:
I don't think this is a  big issue, because you can alias. It's better to have it consistent within the file.

```javascript
import { Family as dfuFamily } from './programmers/dfu-util';
```
would probably fix up any potential namespace clash.